code/datums/elements/drag_pickup.dm

```diff
@@ -0,0 +1,25 @@
+/datum/element/drag_pickup
+	element_flags = ELEMENT_DETACH
+
+/datum/element/drag_pickup/Attach(datum/target)
+	if(isatom(target))
+		if(ismovable(target))
```

Comment:
Remove the isatom entirely--ismovable checks for `/atom/movable`.

Also, change the check to instead check for:

```dm
if (!ismovable(target))
    return ELEMENT_INCOMPATIBLE
```